ocaml/idl/datamodel_host.ml

```diff
@@ -1079,10 +1079,23 @@ let uninstall_ca_certificate =
     ~doc:"Remove a TLS CA certificate from this host."
     ~params:
       [
-        (Ref _host, "host", "The host"); (String, "name", "The certificate name")
+        (Ref _host, "host", "The host")
+      ; (String, "name", "The certificate name")
+      ; ( Bool
+        , "force"
+        , "If true, remove the DB entry even if the file is non-existent"
+        )
       ]
     ~allowed_roles:_R_LOCAL_ROOT_ONLY
-    ~lifecycle:[(Published, "1.290.0", "Uninstall TLS CA certificate")]
+    ~lifecycle:
+      [
+        (Published, "1.290.0", "Uninstall TLS CA certificate")
+      ; ( Changed
+        , "24.29.0"
```

Comment:
@psafont Is this the correct thing to do? This is currently the correct version but it could change before release. Does this just need to be updated again before release or is there some way to auto-fill it? This only happens when you have lifecycle:[] right?